installer/Migration/V5_1_0/Migration.php

```diff
@@ -142,6 +143,15 @@ public function up(): void
         $this->insertSelfReviewWorkflowStates();
         $this->insertReviewListScreenForAdminRole($reviewListScreenId);
         $this->modifyThemeTable();
+
+        $langStringName = "Allows numbers and only + - / ( )";
```

Comment:
Based on spec we should preserve this instead of the other one. The translations also correspond to this
https://twiki.orangehrm.com/bin/view/OrangeHRM/OSAdminOrganizationLocations